doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -472,3 +472,92 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
    usage::
 
       --ssram
+
+----
+
+.. list-table:: Emul Items List For PCI Device Type
+   :header-rows: 1
+
+   * - Emul Type
+     - Description
+
+   * - "xhci"
+     - Usb controller used to support USB3.0 devices, provide downward compatibility
+
+   * - "lpc"
+     - LPC bus used to connect low speed device to cpu, e.g.serial port
+
+   * - "igd-lpc"
+     - Windows graphic driver requires this virtualized LPC device to complete the display function
+
+   * - "ivshmem"
+     - Virtualized PCI device used specifically for shared memory between VMs
+
+   * - "ahci"
+     - Advanced Host Controller Interface used to connect with SATA storage device
+
+   * - "ahci-cd"
+     - Advanced Host Controller Interface used to connect with AT Attachment Packet Interface device
+
+   * - "amd_hostbridge"
+     - Virtualized PCI AMD hostbridge
+
+   * - "hostbridge"
+     - Virtualized PCI hostbridge
+
+   * - "pci_gvt"
+     - Using Intel GVT-g technology
+
+   * - "virtio-blk"
+     - Virtio block type device
+
+   * - "virtio-coreu"
+     - Used for PAVP session management
+
+   * - "virtio-input"
+     - Virtio type device to emulate input device
+
+   * - "virtio-ipu"
+     - Virtio ipu type device
```

Comment:
Is this IPU "Intelligence" (for AI) or "Image" (for graphics) processing unit?

Reply:
like camera device

Reply:
I asked the question assuming you'd put the answer into the documentation not as a comment to this PR.